services/base-service/models/expert/api.js

```diff
@@ -128,11 +163,43 @@ router.route(
 
 // this path is used instead of the defined version in the defaultEsApiGenerator
 router.route(
- '/[a-zA-Z0-9]+/?$'
+  '/:expertId'
 ).get(
+  oapi.path(
+    {
+      "description": "Get an expert by id",
+      "parameters": [
+        {
+          "in": "path",
+          "name": "expertId",
+          "description": "The id of the expert to get",
+          "required": true,
+          "schema": {
+            "type": "string"
+          }
+        }
+      ],
+      "responses": {
+        "200": {
+          "description": "The expert",
+          "content": {
+            "application/json": {
+              "schema": {
+                "$ref": "#/components/schemas/Expert"
+              }
+            }
+          }
+        },
+        "404": {
```

Comment:
In addition, this package also includes components, which maybe can be shared accross all APIS.